src/async/tasks.md

```diff
@@ -51,13 +51,19 @@ async fn main() -> io::Result<()> {
 
 Copy this example into your prepared `src/main.rs` and run it from there.
 
+Try connecting to it with a TCP connection tool like `nc` or `telnet`.
+
+* [Netcat (nc)](https://docs.rs/nc/latest/nc/): Netcat is a versatile networking utility that can be used for various networking tasks, including connecting to network services. 
+
+* [Telnet](https://docs.rs/telnet/latest/telnet/): Telnet is a network protocol used for remote terminal connections to devices over a network. It's a simple way to test network connectivity.
+
```

Comment:
These docs links are still wrong! The linked `nc` crate does not do what the text says it does..

Reply:
sry . i dont know how did i miss that
Fixed them in my latest commit `Fixed links`